.github/workflows/main.yml

```diff
@@ -375,6 +375,49 @@ jobs:
         run: |
           mvn -Dtest=TestSQLiteQPG test
 
+  stonedb:
+    name: DBMS Tests (StoneDB)
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Check disk space
+        run: df . -H
+      - name: Free disk space
+        run: |
+          sudo rm -rf /usr/share/dotnet
```

Comment:
Does the v1.0.3 version of StoneDB also require a huge disk space?

Reply:
no, the v1.0.3 require normal disk space, but let's leave the space clean statement and delete it at the end of the development process?

Reply:
You can leave these statements locally, instead of pushing it to repo. We have to make each commit clean.